WordPress/Classes/Apps/Reader/ReaderTabViewController.swift

```diff
@@ -1,11 +1,30 @@
 import UIKit
 import SwiftUI
+import WordPressUI
 
 final class ReaderTabViewController: UITabBarController {
+    private var menuStore: AnyObject?
```

Comment:
Why declare it as `AnyObject`?

Reply:
I hope to remove it soon. `AnyObject` just to indicate that it's only needed for memory management purposes.